angular-frontend/src/app/components/data-table/table-basic.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, Input, OnInit, SimpleChange } from '@angular/core';
+import { MatTableDataSource } from '@angular/material/table';
+import { GetApiService } from 'src/service/get-api.service';
+import { Attendance } from 'src/Model/Attendance';
+
+@Component({
+  selector: 'table-basic',
+  styleUrls: ['table-basic.css'],
+  templateUrl: 'table-basic.html',
+})
+export class TableBasic implements OnInit {
+  dataSource;
+  displayedColumns: string[] = ['memberId', 'barcode'];
+  selectedValue: string;
+
+  constructor(private api: GetApiService) {}
+
+  ngOnInit() {
+    this.TotalList();
+  }
+
+  TotalList() {
```

Comment:
Naming rant: in TypeScript the best practice is to use camel case for method and variable names. You're using it in most of the rest of the application so having `TotalList` here and a `SeasonArray` variable in a method below is inexplicably inconsistent.